src/main/java/oshi/software/os/windows/nt/CentralProcessor.java

```diff
@@ -136,10 +137,16 @@ public void setFamily(String _family) {
 	 * {@inheritDoc}
 	 */
 	public float getLoad() {
-		throw new UnsupportedOperationException();
-
-		// this is from stack overflow. I don't have windows, so I can't test it.
-		// return Float.valueOf(ExecutingCommand.getFirstAnswer("wmic cpu get loadpercentage"));
+		// this always return whole number value
+		String result = ExecutingCommand.getAnswerAt("wmic cpu get loadpercentage", 2);
+		int load = 0;
+		if (result != null && !result.isEmpty())
+			try {
+				load = Integer.valueOf(result.trim());
+			} catch(NumberFormatException e) {
+				System.err.println("Cannot parse load value: " + result.trim());

```

Comment:
You should re-raise an exception here or let the one above be raised. Applications are getting the wrong result in case of an error, 0.